format_test.go

```diff
@@ -80,6 +82,22 @@ var formatTests = []struct {
 	about: "slice",
 	value: []int{1, 2, 3},
 	want:  "[]int{1, 2, 3}",
+}, {
+	about: "bytes",
+	value: []byte("hello"),
+	want:  `[]uint8("hello")`,
+}, {
+	about: "custom bytes type",
+	value: myBytes("hello"),
+	want:  `quicktest_test.myBytes("hello")`,
+}, {
+	about: "bytes with backquote",
+	value: []byte(`a "b" c`),
+	want:  "[]uint8(`a \"b\" c`)",
+}, {
+	about: "bytes with invalid utf-8",
+	value: []byte("\xff"),
+	want:  "[]uint8{0xff}",
```

Comment:
Add a test with a []byte(nil)?

Reply:
good catch. added the test and fixed the code.